src/rviz_visual_tools.cpp

```diff
@@ -790,16 +792,16 @@ Eigen::Isometry3d RvizVisualTools::getVectorBetweenPoints(const Eigen::Vector3d&
   // Method 1 - TF - works
```

Comment:
@davetcoleman Why is this workaround needed? Is there a unit test for it?

Reply:
No, I was in grad school a the time and did not write enough unit tests. This is an embarrassing section of code :-(

You can see a few lines below my attempt to use Eigen instead did not work. I strongly encourage you to remove these comments and just go with which ever one you prefer.

>  Why is this workaround needed? 

On line 814 you see "Method 2" - uncomment that and see if you can get it to work